eventhubs/client/azure-eventhubs/src/samples/java/SendCustomEventDataList.java

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Jesse's original name: "PublishEventsWithCustomMetadata" is easier to understand the scenario than "SendCustomEventEventDataList"